build/images/katas/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# We use the iqsharp-base image, as that includes
+# the .NET Core SDK, IQ#, and Jupyter Notebook already
+# installed for us.
+FROM mcr.microsoft.com/quantum/iqsharp-base:0.11.2004.2715-beta
+
+# Add metadata indicating that this image is used for the katas.
+ENV IQSHARP_HOSTING_ENV=KATAS_DOCKERFILE
+
+# Make sure the contents of our repo are in ${HOME}
+# Required for mybinder.org
+COPY . ${HOME}
+
+USER root
+RUN chown -R ${USER} ${HOME} && \
+    chmod +x ${HOME}/scripts/*.sh
+
+# Install Python dependencies for the Python visualization and tutorial notebooks
+RUN pip install -I \
```

Comment:
The `-I` switch is new compared to the existing Dockerfile - does it fix the warning reported in https://github.com/microsoft/QuantumKatas/issues/243?

Reply:
No, this is to overwrite previous installed packages.
-I, --ignore-installed
Ignore the installed packages, overwriting them. This can break your system if the existing package is of a different version or was installed with a different package manager!

#243 is still happening.